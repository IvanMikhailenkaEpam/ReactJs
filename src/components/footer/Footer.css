footer {
  position: fixed;
  left: 0;
  bottom: 0;
  width: 100%;
  height: 40px;
  background-color: #3C3B39;
  color: #F85266;
  text-align: left;
  font-weight: bold;
}
.label{
  transform: translateY(-50%); top: 50%; position: relative;
}
