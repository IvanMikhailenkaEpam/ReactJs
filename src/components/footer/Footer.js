import React, {Component} from 'react';
import './Footer.css';

class Footer extends Component {
  render() {
    return (
      <footer className="mt-5">
        <p className="mb-0 ml-4 label">netflixroulette</p>
      </footer>
    );
  }
}

export default Footer;
