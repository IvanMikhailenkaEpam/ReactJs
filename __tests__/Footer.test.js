import React from 'react';
import { shallow } from 'enzyme';
import Footer from "../src/components/footer/Footer";

describe('Footer', () => {
  it('renders Footer correctly', () => {
    const component = shallow(<Footer />);

    expect(component).toMatchSnapshot();
  });
});
