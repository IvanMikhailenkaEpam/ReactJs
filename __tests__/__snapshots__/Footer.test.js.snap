// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer correctly 1`] = `
<footer
  className="mt-5"
>
  <p
    className="mb-0 ml-4 label"
  >
    netflixroulette
  </p>
</footer>
`;
